packages/server/src/scores.test.ts

```diff
@@ -0,0 +1,69 @@
+import { finalScore } from './scores';
```

Comment:
this is not working (of course) we need to setup test environment on the server to make it working with ts-node